internal/platform/storage/couchbase.go

```diff
@@ -0,0 +1,255 @@
+package storage
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"gopkg.in/couchbase/gocb.v1"
+)
+
+const (
+	usersBucketName  = "users"
+	tokensBucketName = "tokens"
+)
+
+var (
+	ErrUserNotFound = errors.New("user not found")
+)
+
+type Storage struct {
+	cluster *gocb.Cluster
+}
+
+func NewCouchbaseStorage(address, username, password string) (*Storage, error) {
```

Comment:
I'm not really happy with this method.

1) I need somehow to initialize basic database structure it's not existing before. That's why there is weird logic about bucket existence and bucket creation
2) I wasn't expected that there might be timeout in bucket creation.. 😢